src/cider/tests/utils/CiderNextgenBenchmarkBase.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <gtest/gtest.h>
+
+#include <string>
+
+#include "tests/utils/CiderNextgenBenchmarkBase.h"
+#include "tests/utils/CiderNextgenQueryRunner.h"
+#include "tests/utils/DuckDbQueryRunner.h"
+#include "tests/utils/QueryArrowDataGenerator.h"
+#include "util/measure.h"
+
+namespace cider::test::util {
+
+void CiderNextgenBenchmarkBase::benchSQL(const std::string& sql) {
```

Comment:
Hmm, why not use some benchmark framework like folly? It covers some basics like warming up, calculate some metrics like sec/op, performance relative ratio.

Reply:
I think we don't need that in such scenario. We just need a time break down, right? it makes no sense to compare performance ratio between duckdb runner and cider runner.